Resources/Patches/save/credits/strings.event

```diff
@@ -1236,7 +1236,7 @@ trapsSprungEnglish:
 String("Traps sprung"); BYTE 0
 
 trapsSprungFrench:
-String("Traps sprung"); BYTE 0
+String("Pi"); BYTE 0x8A; String("ges") d"); BYTE 0xE9; String("clench"); BYTE 0xE9; String("s"); BYTE 0
```

Comment:
You have some mismatched parentheses in there

Reply:
byte 0x8a => BYTE 0xE8 ;)